src/schemas/search.ts

```diff
@@ -8,6 +8,840 @@ import { regex } from './utils';
 const BLACKLIST_STRIPING_REGEX = regex.BLACKLIST_STRIPING_REGEX; 
 
 export default {
+  validation: {
+    type: 'object',
+    strict: true,
+    properties: {
+      clientVersion: {
+        type: 'object',
+        strict: true,
+        properties: {
+          raw: {
+            type: 'string'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/groupby/gb-tracker-client/pullRequest?prid=8314532)
```suggestion
            type: "string"
```